### **DATA EXPLORATION with Python using Pandas & Numpy Libraries**
 Student Names: Krishnendu Adhikary and Shefali Pujara

Enrollment Numbers: 055022, 055044

Group Number: 11


### **Description of DATA**

This dataset provides detailed information on international trade transactions, capturing both import and export activities. It includes comprehensive data on various aspects of trade, making it a valuable resource for business analysis, economic research, and financial modeling.

**Data Source:** https://www.kaggle.com/datasets/chakilamvishwas/imports-exports-15000

**Data Size:** 2001 rows × 16 columns

**Data Type:** Cross-sectional

**Data Dimension:** 16 Variables and 2001 Observations

**Data Variable Types:** Text(11), Integer(3), Decimal(2)

**Data Variable Category:** 

Index: Transaction_ID, Invoice_Number

Nominal Categorical: Country, Product, Import_Export, Category, Port, Customs_Code, Shipping_Method, Supplier, Customer, Payment_Terms

Ordinal Categorical: None

Non-Categorical: Quantity, Value, Weight, Date



### **ANALYSIS**

Basic descriptive and mathematical or statistical analysis

In [7]:
#Importing all required libraries
import os
import pandas as pd
import numpy as np

#Loading the dataset
dataset=pd.read_csv("D:\\MBA materials\\Trimester 1\\DEVP(AMitra)\\Imports_Exports_Dataset.csv")

In [12]:
#Taking the sample dataset
dtsample=dataset.sample(n=2001 , random_state=2244)
dtsample.head(10)

Transaction_ID        Country   Product  \
7345   b28b7985-9865-4611-a189-1f6154999009        Germany     woman   
205    6238d0a0-cf5e-4ea3-a258-4ab368777bab     Costa Rica       day   
5146   0d0c61fd-4700-4eb8-b408-850e0fdc26f4          Chile     group   
5540   0c4a3950-345d-4816-80ed-b229b6b54cde          Ghana      want   
4499   f89d4123-1718-490a-80f6-2b31c5b0bb0d         Bhutan    finish   
4775   3c6a7d4a-384f-4130-a288-137e2889d205       Botswana       eat   
13628  0bdf0b4d-5a1e-4543-8e41-4af89e36eb88          Korea    series   
4111   7d5a1b03-d8ac-4d06-8428-3ad0df038544      Singapore      whom   
1858   5eabd8f7-1782-44e8-8cb7-71b1e922ba0a           Oman      data   
9521   12f671b0-2f22-40b6-89fc-7de9d7e7d52f  Faroe Islands  consumer   

      Import_Export  Quantity    Value        Date     Category  \
7345         Export      4402  2506.89  19-04-2020  Electronics   
205          Import      4638  2119.37  25-09-2023    Machinery   
5146         Import      1696   795.01  18-12-2021    Machinery   
5540         Import       846  3943.25  20-12-2021         Toys   
4499         Import      6191  4624.25  09-06-2020    Furniture   
4775         Export      1215  2537.15  17-04-2021    Machinery   
13628        Export      8544  4153.00  24-06-2024  Electronics   
4111         Import      9844  7277.94  12-04-2024  Electronics   
1858         Export       514  5667.46  28-05-2024    Furniture   
9521         Export      2560  9474.11  08-10-2022     Clothing   

                     Port  Customs_Code   Weight Shipping_Method  \
7345    Lake Ericachester        590842  1862.12             Sea   
205            Amandatown        856709  1457.77            Land   
5146   South Kennethburgh        531811  3899.05             Sea   
5540           Tylershire        903558   686.79             Sea   
4499        Stephanietown        202687   906.43             Sea   
4775             Jonshire        730560  3733.84            Land   
13628       North Zachary        140552  1755.24            Land   
4111           Craighaven        853364   447.79             Air   
1858            Hillville        117188  1359.34             Sea   
9521       Port Peterfurt        473488  1236.42             Sea   

                          Supplier                Customer  Invoice_Number  \
7345   Schneider, Hicks and Potter        Taylor Christian        70000162   
205     Herrera, Chang and Farrell            Brittany Lee        88203466   
5146                     Pratt Ltd      Thomas Hartman DVM        24718885   
5540                Hughes-Sweeney          Michelle Kirby        93019204   
4499                 Salazar Group           Anthony Wiley        31583917   
4775           Moore, Blake and Li             Pamela Diaz        96813919   
13628                  Patel Group           Sara Erickson        51300134   
4111                   Morris-Boyd          Rachael Warner        23648012   
1858    Barrett, Hughes and Hudson            Austin Lewis        75486163   
9521                    Arnold Inc  Mr. Thomas Mcbride DVM        60059429   

          Payment_Terms  
7345             Net 60  
205             Prepaid  
5146             Net 30  
5540             Net 30  
4499            Prepaid  
4775   Cash on Delivery  
13628  Cash on Delivery  
4111             Net 60  
1858             Net 60  
9521            Prepaid

**1. What is the minimum quantity shipped in any transaction?**

In [206]:
Min= dtsample['Quantity'].min()
print('The minimum quantity shipped in any transaction:', Min)

The minimum quantity shipped in any transaction: 1


**2. What is the maximum weight of goods shipped across all transactions?**

In [207]:
Max= dtsample['Weight'].max()
print('The maximum weight of goods shipped across all transactions:', Max)

The maximum weight of goods shipped across all transactions: 4995.7


**3. What is the average value of exports and imports?**

In [208]:
avg= dtsample['Value'].mean()
print('The average value of exports and imports:', avg)

The average value of exports and imports: 5015.6328035982015


**4. What is the median quantity of products in the transactions?**

In [209]:
med= dtsample['Quantity'].median()
print('The median quantity of products in the transactions:', med)

The median quantity of products in the transactions: 4871.0


**5. What is the most common transaction values in this dataset?**

In [14]:
mode= dtsample['Value'].mode()
print('The most common transaction values in this dataset:')
print(mode)

The most common transaction values in this dataset:
0     241.83
1    4338.20
2    9295.66
Name: Value, dtype: float64


**6. What is the 75th percentile of transaction values in this dataset?**

In [211]:
q3= dtsample['Value'].quantile(0.75)
print('The 75th percentile of transaction values in this dataset:', q3)

The 75th percentile of transaction values in this dataset: 7459.23


**7. What is the range of product weights in the dataset?**

In [212]:
range= (dtsample['Weight'].max() - dtsample['Weight'].min())
print('The range of product weights in the dataset:', range)

The range of product weights in the dataset: 4994.179999999999


**8. What is the standard deviation of the quantity of products across transactions?**

In [213]:
sd= dtsample['Quantity'].std()
print('The standard deviation of the quantity of products across transactions:', sd)

The standard deviation of the quantity of products across transactions: 2846.1893173333433


**9. Is the distribution of the total transaction value skewed?**

In [214]:
skew= dtsample['Value'].skew()
if skew>0:
    print('The value of skewness:', skew)
    print('Yes. The distribution of the total transaction value is positively skewed.')
elif skew==0:
    print('The value of skewness:', skew)
    print('No. The distribution of the total transaction value is not skewed.')
else:
    print('The value of skewness:', skew)
    print('Yes. The distribution of the total transaction value is negatively skewed.')

The value of skewness: 0.049914786003225936
Yes. The distribution of the total transaction value is positively skewed.


**10. How peaked or flat is the distribution of the transaction values?**

In [215]:
kurt= dtsample['Value'].kurtosis()
if kurt>3:
    print('The value of Kurtosis:', kurt)
    print('Type of Kurtosis: Leptokurtic')
elif kurt==3:
    print('The value of Kurtosis:', kurt)
    print('Type of Kurtosis: Mesokurtic')
else:
    print('The value of Kurtosis:', kurt)
    print('Type of Kurtosis: Platykurtic')

The value of Kurtosis: -1.2128780317794574
Type of Kurtosis: Platykurtic


**11. Is there a correlation between the quantity of goods and their transaction value?**

In [216]:
corr = dtsample['Quantity'].corr(dtsample['Value'])
if corr>0:
    print('The value of correlation:', corr)
    print('Type of Correlation: Positive')
elif corr==0:
    print('The value of correlation:', corr)
    print('Type of Correlation: No Correlation')
else:
    print('The value of correlation:', corr)
    print('Type of Correlation: Negative')

The value of correlation: -0.04082038843265241
Type of Correlation: Negative


**12. What is the coefficient of variation for transaction values?**

In [217]:
coef = dtsample['Value'].std() / dtsample['Value'].mean()
print('Coefficient of Variation of Transaction Values:', coef)

Coefficient of Variation of Transaction Values: 0.5717370617535402


**13. What is the 95% confidence interval for the mean quantity of products shipped?**

In [218]:
import scipy.stats as stats

# Calculating the 95% confidence interval for the mean of quantity
confidence_level = 0.95
sample_size = len(dtsample['Quantity'])
sample_mean = dtsample['Quantity'].mean()
sample_std = dtsample['Quantity'].std()

z_critical = stats.norm.ppf(q = 0.975)  # Z-value for 95% confidence interval
margin_of_error = z_critical * (sample_std / (sample_size**0.5))

confidence_interval = (sample_mean - margin_of_error, sample_mean + margin_of_error)
print('95% Confidence Interval for Mean Quantity:' , confidence_interval)

95% Confidence Interval for Mean Quantity: (4803.557086420671, 5052.969650211013)


**14. How many unique countries are involved in these transactions?**

In [219]:
unique_countries = dtsample['Country'].nunique()
print('Number of Unique Countries:' , unique_countries)

Number of Unique Countries: 243


**15. Which product is traded most frequently?**

In [220]:
most_frequent_product = dtsample['Product'].mode()[0]
print('Most Frequent Product:', most_frequent_product)

Most Frequent Product: bank


**16. What proportion of the transactions are exports vs. imports?**

In [221]:
proportion_exports = (dtsample['Import_Export'] == 'Export').mean()
proportion_imports = (dtsample['Import_Export'] == 'Import').mean()
prop_exp = round(proportion_exports, 3)
prop_imp = round(proportion_imports, 3)
print(f"Proportion of Exports: {prop_exp * 100}%"),
print(f"Proportion of Imports: {prop_imp * 100}%")

Proportion of Exports: 48.5%
Proportion of Imports: 51.5%


**17. Which shipping method was used the least across all transactions?**

In [222]:
least_used_shipping = dtsample['Shipping_Method'].value_counts().idxmin()
print('Least Used Shipping Method:', least_used_shipping)

Least Used Shipping Method: Land


**18. Which country has the highest number of transactions?**

In [223]:
country_with_max_transactions = dtsample['Country'].value_counts().idxmax()
print('Country with the Most Transactions:', country_with_max_transactions)

Country with the Most Transactions: Congo


**19. What is the most common shipping method used for transactions?**

In [224]:
most_common_shipping = dtsample['Shipping_Method'].mode()[0]
print('Most Common Shipping Method:', most_common_shipping)

Most Common Shipping Method: Air


**20. Rank the Top 10 countries based on the total value of goods traded.**

In [225]:
ranked_countries = dtsample.groupby('Country')['Value'].sum().sort_values(ascending=False)
top10= ranked_countries.head(10)
print(f"Top 10 Ranked Countries by Total Value: \n{top10}")

Top 10 Ranked Countries by Total Value: 
Country
Congo                               128801.65
New Zealand                          96808.21
Papua New Guinea                     84352.79
Costa Rica                           82931.08
Korea                                80972.61
Romania                              79649.01
Saint Vincent and the Grenadines     79219.23
Comoros                              79127.12
Indonesia                            77978.79
Vietnam                              77446.54
Name: Value, dtype: float64


**21. Is there a relationship between Country and Category of products shipped?**

In [226]:
from scipy.stats import chi2_contingency

# Create a contingency table
contingency_table = pd.crosstab(dtsample['Country'], dtsample['Category'])

# Perform Chi-Square Test of Independence
chi2_stat, p_value, dof, expected = chi2_contingency(contingency_table)

if p_value < 0.05:
    print(f"p-value: {p_value}")
    print("Reject the null hypothesis: There is a significant relationship between country and product category.")
else:
    print(f"p-value: {p_value}")
    print("Fail to reject the null hypothesis: No significant relationship between country and product category.")

p-value: 0.19102098265359746
Fail to reject the null hypothesis: No significant relationship between country and product category.


**22. Is the average quantity of products shipped significantly different from 5000 units?**

In [227]:
from scipy import stats

# Define the null hypothesis: mean quantity = 5000
t_stat, p_value = stats.ttest_1samp(dtsample['Quantity'], 5000)

if p_value < 0.05:
    print(f"p-value: {p_value}")
    print("Reject the null hypothesis: The average quantity is significantly different from 5000 units.")
else:
    print(f"p-value: {p_value}")
    print("Fail to reject the null hypothesis: The average quantity is not significantly different from 5000 units.")

p-value: 0.25968372632163167
Fail to reject the null hypothesis: The average quantity is not significantly different from 5000 units.


**23. Is there a significant difference in transaction values across different shipping methods?**

In [228]:
# Perform one-way ANOVA across different shipping methods
anova_stat, anova_p_value = stats.f_oneway(
    dtsample[dtsample['Shipping_Method'] == 'Sea']['Value'],
    dtsample[dtsample['Shipping_Method'] == 'Land']['Value'],
    dtsample[dtsample['Shipping_Method'] == 'Air']['Value']
)

if anova_p_value < 0.05:
    print(f"p-value: {anova_p_value}")
    print("Reject the null hypothesis: There is a significant difference in transaction values across shipping methods.")
else:
    print(f"p-value: {anova_p_value}")
    print("Fail to reject the null hypothesis: No significant difference in transaction values across shipping methods.")

p-value: 0.3310634793545532
Fail to reject the null hypothesis: No significant difference in transaction values across shipping methods.


**24. Is the variance in Quantity significantly different between imports and exports?**

In [229]:
from scipy.stats import levene

# Separate data into imports and exports
import_data = dtsample[dtsample['Import_Export'] == 'Import']['Quantity']
export_data = dtsample[dtsample['Import_Export'] == 'Export']['Quantity']

# Perform Levene's test for equal variances
levene_stat, p_value = levene(import_data, export_data)

# Evaluate result
if p_value < 0.05:
    print(f"The p-value is {p_value}. The variance in Quantity is significantly different between imports and exports.")
else:
    print(f"The p-value is {p_value}. There is no significant difference in the variance of Quantity between imports and exports.")

The p-value is 0.2241854127277818. There is no significant difference in the variance of Quantity between imports and exports.


**25. Is there a significant difference in the proportion of imports vs. exports?**

In [230]:
from statsmodels.stats.proportion import proportions_ztest

# Count imports and exports
import_count = sum(dtsample['Import_Export'] == 'Import')
export_count = sum(dtsample['Import_Export'] == 'Export')
total_count = len(dtsample)

# Perform z-test for proportions
stat, p_value = proportions_ztest([import_count, export_count], [total_count, total_count])

# Evaluate result
if p_value < 0.05:
    print(f"The p-value is {p_value}. There is a significant difference in the proportion of imports vs. exports.")
else:
    print(f"The p-value is {p_value}. There is no significant difference in the proportion of imports vs. exports.")

The p-value is 0.06214250909501448. There is no significant difference in the proportion of imports vs. exports.


**26. Is the correlation between Quantity and Value significant?**

In [231]:
from scipy.stats import pearsonr

# Calculate Pearson correlation between Quantity and Value
correlation, p_value = pearsonr(dtsample['Quantity'], dtsample['Value'])

# Evaluate result
if p_value < 0.05:
    print(f"The p-value is {p_value}. The correlation between Quantity and Value is significant.")
else:
    print(f"The p-value is {p_value}. The correlation between Quantity and Value is not significant.")

The p-value is 0.06790759636643746. The correlation between Quantity and Value is not significant.


**27. Is the distribution of Value normal?**

In [232]:
from scipy.stats import shapiro

# Perform Shapiro-Wilk test for normality on Value
stat, p_value = shapiro(dtsample['Value'])

# Evaluate result
if p_value < 0.05:
    print(f"The p-value is {p_value}. The distribution of Value is not normal.")
else:
    print(f"The p-value is {p_value}. The distribution of Value is normal.")

The p-value is 7.31824079278583e-25. The distribution of Value is not normal.


**28. How well can we predict Value based on Quantity and Weight?**

In [28]:
import statsmodels.api as sm

# Define dependent (y) and independent variables (X)
X = dtsample[['Quantity', 'Weight']]
y = dtsample['Value']

# Add constant to the model (intercept)
X = sm.add_constant(X)

# Fit the linear regression model
model = sm.OLS(y, X).fit()

# Evaluate result based on p-values
if model.pvalues['Quantity'] < 0.05 and model.pvalues['Weight'] < 0.05:
    print("Both Quantity and Weight are significant predictors of Value.")
elif model.pvalues['Quantity'] < 0.05:
    print("Only Quantity is a significant predictor of Value.")
elif model.pvalues['Weight'] < 0.05:
    print("Only Weight is a significant predictor of Value.")
else:
    print("Neither Quantity nor Weight are significant predictors of Value.")

Neither Quantity nor Weight are significant predictors of Value.


**29. Can we predict whether a transaction is an Import or Export based on Quantity and Weight?**

In [234]:
from sklearn.linear_model import LogisticRegression
from sklearn.model_selection import train_test_split
from sklearn.metrics import accuracy_score

# Convert 'Import_Export' to binary (0 = Import, 1 = Export)
dtsample['Import_Export_binary'] = dtsample['Import_Export'].map({'Import': 0, 'Export': 1})

# Define independent (X) and dependent variables (y)
X = dtsample[['Quantity', 'Weight']]
y = dtsample['Import_Export_binary']

# Split data into training and test sets
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42)

# Fit the logistic regression model
logreg = LogisticRegression()
logreg.fit(X_train, y_train)

# Make predictions
y_pred = logreg.predict(X_test)
accuracy = accuracy_score(y_test, y_pred)

# Evaluate result based on accuracy
if accuracy > 0.7:
    print(f"The logistic regression model predicts Import/Export with {accuracy*100:.2f}% accuracy, indicating a good model.")
else:
    print(f"The logistic regression model has an accuracy of {accuracy*100:.2f}%, indicating it may not be a strong predictor.")

The logistic regression model has an accuracy of 49.63%, indicating it may not be a strong predictor.


**FINDINGS AND INFERENCES**

 **NON-CATEGORICAL DATA**



**1.	Minimum - What is the minimum quantity shipped in any transaction?**

o	Observation: The minimum quantity shipped could be a small number, e.g., 1 unit

o	Managerial Insight: The company should assess whether low-volume shipments are cost-effective, especially for exports. Small orders may not justify the shipping and operational costs, especially when using costly shipping methods like air freigh.


**2. Maximum - What is the maximum weight of goods shipped across all transactions?**

o	Observation: The maximum weight is around 499.5 kg.

o	Managerial Insight: Understanding the upper limit of shipment weight is essential for logistics planning. Heavy shipments may require special transport arrangements (e.g., sea freight) or contracts with shipping companies to reduce costs.


**3.	Mean - What is the average value of exports and imports?**

o	Observation: The average transaction value is approximately $5015.

o	Managerial Insight: The average transaction value helps management assess revenue expectations per transaction. It allows the company to identify whether high or low-value transactions are dominating their operations, informing pricing strategies.


**4.	Median - What is the median quantity of products in the transactions?**

 o	Observation: The median quantity is 4,871 units.

 o	Managerial Insight: The median quantity helps in understanding the typical shipment size, helping managers better predict warehouse and transport needs. If the median is significantly lower than the mean, it could indicate a few very large orders skewing the data.


**5.	Mode - What is the most common transaction values in this dataset?**

o	Observation: Common transaction values in the data set are 241.83 , 4338.20 , 9295.66.

o	Managerial Insight: This indicates the typical transaction value in the dataset, which can help in defining standard pricing tiers or discounts for customers. Understanding the mode also assists in inventory and financial planning.


**6.	Percentile - What is the 75th percentile of transaction values?**

o	Observation: The 75th percentile value is $7459.23.

o	Managerial Insight: This helps to identify the high-value transactions that constitute the top 25% of the dataset. Focusing on this group could help target key accounts or customers that bring in more revenue.


**7.	Range - What is the range of product weights in the dataset?**

o	Observation: The weight ranges from 1.52 kg to 4995.7 kg.

o	Managerial Insight: A wide range suggests a diverse product portfolio, and different shipping methods might be more suitable for different products. It helps in assessing warehouse space and determining the most cost-effective shipping methods for each product category.


**8.	Standard Deviation - What is the standard deviation of the quantity of products across transactions?**

o	Observation: A high standard deviation indicates a large variation in product quantities across transactions.

o	Managerial Insight: A high variation in order sizes implies unpredictable demand patterns, making it difficult to plan inventory and logistics. More flexible supply chain strategies might be needed, such as scalable warehousing and dynamic pricing.


**9. Skewness - Is the distribution of the total transaction value skewed?**

o	Observation: If the skewness is positive, most transactions are smaller, with a few large ones driving the overall value. If the skewness is negative, most transactions are larger, with a few smaller ones pulling the overall value down.

o	Managerial Insight: Skewed distributions suggest that most customers make small orders, with a few placing large orders. Understanding this can help management create different sales strategies, such as offering promotions for small customers and personalized services for large customers.


**10.	Kurtosis - How peaked or flat is the distribution of the transaction values?**

o	Observation: A platykurtic distribution (low kurtosis) indicates that transaction values are more spread out, with fewer values close to the mean and a wider range of transactions.

o	Managerial Insight: A high kurtosis indicates a focus on a specific order size, which can help in standardizing operations. Lower kurtosis suggests more variability, requiring flexibility in managing different customer needs.


**11.	Correlation - Is there a correlation between the quantity of goods and their transaction value?**

o	Observation: A negative correlation would indicate that as the quantity increases, the transaction value tends to decrease.

o	Managerial Insight: A negative correlation suggests that customers may be purchasing larger quantities at lower prices, potentially due to bulk discounts or lower-cost items. Management can consider revisiting pricing strategies, ensuring that larger orders still maintain profitability, or offering upsell opportunities on higher-margin products alongside bulk purchases.


**12 Coefficient of Variation - What is the coefficient of variation for transaction values?**

o	Observation: A high coefficient of variation means high variability in transaction values relative to the mean.

o	Managerial Insight: High variability suggests unpredictable revenue streams, which may require more robust financial planning. It may also indicate the need for more targeted marketing to smooth revenue streams.


**13.	Confidence Interval - What is the 95% confidence interval for the mean quantity of products shipped?**

o	Observation: The 95% confidence interval provides a range around the mean, e.g., between 4803 and 5052 units.

o	Managerial Insight: This gives management an estimate of the reliability of the mean. If the range is too wide, it indicates higher uncertainty in shipment volumes, which may impact operational planning.


**CATEGORICAL DATA**

**14.	Count - How many unique countries are involved in these transactions?**

o	Observation: There are 243 unique countries.

o	Managerial Insight: This helps the company understand its market reach. If the number of unique countries is low, the company may consider expanding into new regions to grow the business.


**15.	Frequency - Which product is traded most frequently?**

o	Observation: Bank is the most frequently traded product.

o	Managerial Insight: Knowing the most popular product category helps in optimizing inventory and production planning. The company may want to focus marketing efforts on popular products to further increase sales.


**16.	Proportion - What proportion of the transactions are exports vs. imports?**

o	Observation: 48.5% are exports and 51.5% are imports.

o	Managerial Insight -This scenario suggests a trade imbalance, where management could explore boosting export opportunities through enhanced marketing, competitive pricing, or product adaptation to international markets. Simultaneously, the company might need to assess the sustainability of high import reliance, exploring local sourcing alternatives or optimizing import costs to improve profitability. 


**17. Minimum - Which shipping method was used the least across all transactions?**

o	Observation: Land the least used.


o	Managerial Insight: If certain shipping methods are underutilized, the company might explore cost-benefit analyses for these methods or discontinue them if they aren't efficient.
.


**18.	Maximum - Which country has the highest number of transactions?**

o	Observation: Congo has the most transactions.


o	Managerial Insight: Knowing which country has the highest transaction volume helps target marketing and sales efforts. Expanding relationships in high-transaction countries can increase market share.


**19.Mode-What is the most common shipping method used for transactions?**

o	Observation: Air is the most common shipping method, that saves time.


o	Managerial Insight: Understanding the most common shipping method helps company negotiate better rates and improve logistically efficienc for that mode of transport.

**20.	Rank - Rank the countries based on the total value of goods traded?**

o	Observation: For instance, Congo , is rank highest, followed by New Zealand.

o	Managerial Insight: This ranking can guide the allocation of resources. High-value countries may warrant more marketing focus or better customer relationship management.



**21. Is there a relationship between country and category of products shipped?**

o Observation: The p-value of 0.191 indicates no significant relationship between country and product category. Product distribution across countries appears uniform, without meaningful differences.

o Managerial InsightA standardized product strategy across countries is feasible, reducing the need for country specific product adjustments and enabling more efficient global operations.t.

**STATISTICAL TESTS**


**22.	Test of Mean (t-test) - Is the average quantity of products shipped significantly not different from 5000 units?**

o	Observation: The t-test shows that the mean is significantly not different from 5000 units.

o	Managerial Insight: If the mean differs significantly, it could suggest that the company should adjust production and inventory levels based on actual demand rather than assumptions.


**23.	ANOVA - Is there a significant difference in transaction value across different shipping methods?**

o	Observation: ANOVA may show that transaction values are significantly different across shipping methods.No significant difference is seen in this case.

o	Managerial Insight: If shipping methods influence transaction values, optimizing or selecting the right shipping method could increase profitability.


**24.	Levene’s Test - Is the variance in Quantity significantly different between imports and exports?**

o	Observation: If the test shows a significant difference, it indicates higher variability in one group (e.g., exports). In this case, there is no significant difference observed.


o	Managerial Insight: The company may need different strategies for managing imports and exports, like separate inventory management systems.


**25.	Test of Proportion (z-test) - Is there a significant difference in the proportion of imports vs. exports?**

o	Observation: If there is a significant difference, e.g., more exports than imports.In our case no significance difference.

o	Managerial Insight: A significantly higher export proportion might indicate untapped potential for imports, or a focus on growing international markets.


**26.	Test of Correlation (t-test) - Is the correlation between Quantity and Value significant?**

o	Observation: A significant positive correlation suggests that transaction value increases as quantity increases.No significant correlation between quantity and value.

o	Managerial Insight: The company could incentivize bulk purchases, offering discounts for larger quantities, as it aligns with higher transaction values.


**27.	Shapiro-Wilk Test - Is the distribution of Value normal?**

o	Observation: If the data is non-normal, the company might need to use non-parametric methods for analysis.

o	Managerial Insight: This insight ensures accurate decision-making based on appropriate statistical methods, guiding strategy formulation based on reliable data.


**REGRESSION AND LOGISTIC REGRESSION**

**28.	Regression - How well can we predict Value based on Quantity and Weight?**

o	Observation: A significant regression model could explain 70% of the variance in Value. As the p values of both Quantity and Weight are greater than 0.05, neither Quantity nor Weight are significant predictors of Value.

o	Managerial Insight: Predicting transaction values based on key variables helps in pricing decisions and optimizing sales strategies for high-value transactions.


**29.	Logistic Regression - Can we predict whether a transaction is an Import or Export based on Quantity and Weight?**

o	Observation: The logistic regression model has an accuracy of 49.63%, indicating it may not be a strong predictor.

o	Managerial Insight: Understanding the factors driving import or export transactions allows the company to forecast future business and tailor logistical strategies accordingly.
